.github/workflows/build-and-push.yml

```diff
@@ -58,7 +58,7 @@ jobs:
           registry_username: ${{ secrets[matrix.quayio_username] }}
           registry_token: ${{ secrets[matrix.quayio_token] }}
           dockerfile: ${{ matrix.dockerfile }}
-          docker_context: ${{ matrix.dockerfile_path }}
```

Comment:
Isn't it the same as `docker_context: '.'`?

Reply:
Good point, I would expect that to be the same. Updated.